metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -613,4 +613,4 @@ def _fill_buf(self):
         events = self._get_events()
         for event in events:
             self._buf.append(event['message'])
-            self._pos = event['timestamp']
\ No newline at end of file
+            self._pos = event['timestamp']
```

Comment:
Given that this file has no changes, you can remove this file from the PR.

Reply:
Got it. There was an extra line at the bottom that made this file come up on the PR. Fixed this.